CCDB/src/CcdbApi.cxx

```diff
@@ -119,6 +119,8 @@ void CcdbApi::init(std::string const& host)
     snapshotReport += ')';
   }
 
+  mNeedAlienToken = host == "http://o2-ccdb.internal" || host == "http://localhost:8084" || host == "http://127.0.0.1:8084";
```

Comment:
Hi @davidrohr  isn't the logic here wrong? Isn't there a `!` missing around all the host checks?